ssl/ssl_conf.c

```diff
@@ -543,6 +549,7 @@ static const ssl_conf_cmd_tbl ssl_conf_cmds[] = {
     SSL_CONF_CMD_STRING(SignatureAlgorithms, "sigalgs", 0),
     SSL_CONF_CMD_STRING(ClientSignatureAlgorithms, "client_sigalgs", 0),
     SSL_CONF_CMD_STRING(Curves, "curves", 0),
```

Comment:
Change it to `SSL_CONF_CMD_STRING(Groups, "curves", 0),` rather than defining another function that just passes arguments...